content/references/protocol-reference/transactions/transaction-results/tec-codes.md

```diff
@@ -43,7 +49,9 @@ For the most part, transactions with `tec` codes take no action other than to de
 | `tecNO_LINE_REDUNDANT`     | 127   | The transaction failed because it tried to set a trust line to its default state, but the trust line did not exist. |
 | `tecNO_PERMISSION`         | 139   | The sender does not have permission to do this operation. For example, the [EscrowFinish transaction][] tried to release a held payment before its `FinishAfter` time, someone tried to use [PaymentChannelFund][] on a channel the sender does not own, or a [Payment][] tried to deliver funds to an account with the "DepositAuth" flag enabled. |
 | `tecNO_REGULAR_KEY`        | 131   | The [AccountSet transaction][] tried to disable the master key, but the account does not have another way to [authorize transactions](transaction-basics.html#authorizing-transactions). If [multi-signing](multi-signing.html) is enabled, this code is deprecated and `tecNO_ALTERNATIVE_KEY` is used instead. |
+| `tecNO_SUITABLE_NFTOKEN_PAGE` | 154 | ***TODO*** |
```

Comment:
The account is attempting to mint or acquire an `NFTokenID` that cannot be held by any of the account's `NFToken` directory pages.  This circumstance can be engineered with some effort, but should never occur through simple minting and/or acquisition of `NFTokens`.